python/lsst/ts/mtdome/schema/lcs_status.py

```diff
@@ -19,18 +19,22 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+__all__: list[str] = []
```

Comment:
Given that you have no symbols to import, does it really make sense to do `from lcs_status import *` in `__init__.py` (of this and most of the modules in `schema/`)? I worry it may confuse users, so I lean towards using `import lcs_status` instead. The down side is that doesn't do the right thing if in the future somebody adds symbols to one or more of these modules that you do want hoisted.

Reply:
It turns out to be even simpler: the `__all__` can be removed from all schema files and needs to stay in `registry.py`. Then all `from .xxxxx import *` can be removed, except again for the registry file. The schemas are only supposed to be available via the schema registry and this actually makes that stronger.

Reply:
That sounds great, but...how do these modules get loaded so that they can register their contents? I assumed that was why `__init__.py` was importing them.

Reply:
They register themselves. Then the CSC and MockController code import the registry and they have all schemas at their disposal. You taught me that trick :)

Reply:
I am still confused. I don't understand how they can register themselves without somebody importing these modules. (It's not meant to block merging; I am just asking for my own curiosity).

Reply:
See for instance these lines:

https://github.com/lsst-ts/ts_mtdome/blob/develop/python/lsst/ts/mtdome/schema/amcs_status.py#L24-L26

No need for `__all__` in that file because nothing is exported. The code simply adds the contents of the schema file to the registry and the registry does get exported.